### DISCUSSION TITLE
How does the new pricing works?

### DISCUSSION THREAD

**Discussion by Sulomus** [FlowFuse/flowfuse#1733] [Q&A] [answered]
Today I received an email informing me that the pricing will be changed from 15$ per month per PROJECT to 15$ per month per DEVICE.

I wanted to ask if this is change is permanent or if there is any way that we could use the old pricing?
+ will there be 15$ per device only or also + project?

**Comment by ZJvandeWeg** [comment, marked as answer]
@Sulomus Thanks for reaching out. The change is what we expect the model to be for the foreseeable future. 

A cloud instance can be suspend at any time and your devices will continue to run. In short; you'll be charged for each running Node-RED runtime.
